src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -758,15 +758,12 @@ def find_using_common_tenant(self, username, credential=None):
             try:
                 subscriptions = self.find_using_specific_tenant(tenant_id, specific_tenant_credential)
             except AuthenticationError as ex:
-                # because user creds went through the 'common' tenant, the error here must be
-                # tenant specific, like the account was disabled. For such errors, we will continue
-                # with other tenants.
-                msg = ex.error_msg
-                if 'AADSTS50076' in msg:
-                    # The tenant requires MFA and can't be accessed with home tenant's refresh token
-                    mfa_tenants.append(t)
-                else:
-                    logger.warning("Failed to authenticate %s due to error '%s'", t.tenant_id_name, ex)
```

Comment:
The effective change of this PR is pretty small: removing `if` and `else` and combining their corresponding statements.